app/bundles/CoreBundle/Command/ModeratedCommand.php

```
@@ -135,7 +135,9 @@ protected function completeRun(): void
         }
 
         // Attempt to keep things tidy
-        @unlink($this->lockFile);
+        if ($this->lockFile && file_exists($this->lockFile)) {
```

I'm wondering when the `$this->lockFile` might not exist. Also, do you think we should throw an exception in that case?

---

`$lockFile` is intentionally only created in PID mode. For other locking modes, cleanup is handled via `$this->lock->release()`, so `$lockFile` being unset is expected and not an error.